### DISCUSSION TITLE
Developing Pokedex in JavaScript

### DISCUSSION THREAD

**Discussion by RyanLH95** [community/community#65596] [Programming Help]
### Select Topic Area

Product Feedback

### Body

Hello,

I am in need of advice. I have developed a pokedex using HTML and CSS and I am ready to start editing it in JavaScript. Can I just have some advice on the types of the pokemon showing up on the type-screens that I have created please?

I have created a divs in HTML called id="primary-type" and id="secondary-type" and was going use these to call each type when called. Would this work or would I also have to add the types individually in JavaScript? (lines 149 - 185 contain the divs specified)
Thank you.

pokedex link: https://ryanlh95.github.io/pokedex/

Here is the link to my html, css and javascript files:

https://github.com/RyanLH95/pokedex/blob/main/index.html

https://github.com/RyanLH95/pokedex/blob/main/style.css

https://github.com/RyanLH95/pokedex/blob/main/app.js

**Comment by rumon5h** [comment]
I have already created this pokemon website you can get idea from this [repository  ](https://github.com/rumon5h/pokemon.git)

**Comment by ghostinhershell** [comment]
Thanks for posting in the GitHub Community, @RyanLH95! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information. Cheers!

**Comment by jamesweber7** [comment]
I'm not entirely sure if this answers your question, but if you want to access each of the types in the type-screens you can use:
```
let typeOneEntries = document.getElementById('type-one');
let typeDivs = [...typeOneEntries.children]; // array of type one entry divs
let types = typeDivs.map(div => div.innerText); // array of each div's inner text
// ... repeat for type two
```
